flutter_reactive_ble_platform_interface/lib/src/reactive_ble_platform_interface.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter_reactive_ble_platform_interface/src/plugin_controller.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
```

Comment:
Is this a standard package? I can't find the content in the pull-request, but also not on pub.dev.

Reply:
I guess, it's this: https://github.com/flutter/plugins/tree/master/packages/plugin_platform_interface

Reply:
yes indeed it is a general recommendation by google "Make sure you use package:plugin_platform_interface to enforce that implementers of your platform interface use extends rather than implements"